packages/registry/docs/conceptual-guides/add-on-styles-loader.md

```diff
@@ -0,0 +1,49 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation of the add-ons styles loader in @plone/registry"
+    "property=og:description": "An explanation of the add-ons styles loader in @plone/registry"
+    "property=og:title": "Add-ons styles loader"
+    "keywords": "@plone/registry, registry, add-ons, loader"
+---
+
+# Add-ons styles loader
+
+The `@plone/registry` compatible add-ons are able to declare styles that should be loaded by the app.
+This is achieved by creating a folder {file}`styles` with a {file}`main.css` file that serves as entry point.
```

Comment:
Please verify where the file should be placed. I guessed. Also we can merge steps of creating a directory and file into just a file. I can do that via command line or in PyCharm's file browser.

```suggestion
To do so, create a file {file}`styles/main.css` at the root of your project which serves as the entry point.
```